src/pages/token/rewards/RewardsPage.tsx

```diff
@@ -75,6 +81,11 @@ const RewardsPage = () => {
     </$LegalDisclaimer>
   );
 
+  useEffect(() => {
+    // Initialize daily data for rewards table + chart
```

Comment:
followup -- maybe we can set the default in abacus to fetch DAILY instead so we don't need to do this?

Reply:
hmmm abacus _should_ be doing that (all the defaults are set to daily) but it doesn't seem to work T.T Will look more into it later!